Structure_Engine/Modify/SetRebarIntent.cs

```diff
@@ -38,17 +38,17 @@ public static partial class Modify
         /**** Public Methods                            ****/
         /***************************************************/
 
-        [Description("Sets Reinforcement to a ConcreteSection. Any previous Reinforcement will be overwritten.")]
-        [Input("section", "The ConcreteSection to set Reinforcement to.")]
-        [Input("reinforcement", "The collection of Reinforcement to set to the ConcreteSection.")]
-        [Output("concSection", "The ConcreteSection with new Reinforcement.")]
-        public static ConcreteSection SetReinforcement(this ConcreteSection section, IEnumerable<IBarReinforcement> reinforcement)
+        [Description("Sets RebarIntent to a ConcreteSection. Any previous RebarIntent will be overwritten.")]
+        [Input("section", "The ConcreteSection to set RebarIntent to.")]
+        [Input("rebarIntent", "The RebarIntent to set to the ConcreteSection.")]
+        [Output("concSection", "The ConcreteSection with new RebarIntent.")]
+        public static ConcreteSection SetRebarIntent(this ConcreteSection section, BarRebarIntent rebarIntent)
```

Comment:
As above. WOuld keep method signature intact, but change the behaviour to setting the reinforcement of the inner RebarIntent